File: CodeFreak1/ViewModel/CompleteSubmissionViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace CodeFreak1.ViewModel
{
    public class CompleteSubmissionViewModel:RequestStatus
    {
        public SubmissionViewModel Submission { get; set; }
        public UsersViewModel User { get; set; }
        public ProblemViewModel Problem { get; set; }
        public ProgrammingLanguageViewModel Language { get; set; }
        public FileViewModel UserImage { get; set; }
    }
}

File: CodeFreak1/ClientApp/src/app/problem/dtos/complete-submission-view-model.ts
import { SubmissionViewModel } from "./submission-view-model";
import { UsersViewModel } from "../../Security/Dtos/users-view-model";
import { ProblemViewModel } from "./problem-view-model";
import { ProgrammingLanguageViewModel } from "../../programming-language/dtos/programming-language-view-model";
import { RequestStatus } from "../../Security/Dtos/request-status";
import { FileViewModel } from "../../Security/Dtos/file-view-model";

export class CompleteSubmissionViewModel extends RequestStatus {
  public Submission: SubmissionViewModel;
  public User: UsersViewModel;
  public Problem: ProblemViewModel;
  public Language: ProgrammingLanguageViewModel;
  public UserImage: FileViewModel;
}
